OWNERS

```diff
@@ -0,0 +1,6 @@
+assignees:
+- migmartri
```

Comment:
In the Kubernetes repo, the [OWNERS file](https://github.com/kubernetes/charts/blob/master/OWNERS) has a different indentation. I'm not sure if this kind of files are standardized